app/views/posts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<%=stylesheet_link_tag('post')%>
+<div class="main-post">
+  <div>
+    <i class="fa-brands fa-square-x-twitter"></i>
+  </div>
+  <div class="post-name">
+    <h3> User name: <%= @user.name %> </h3>
+    <p> Number of posts: <%= @user.posts_counter %> </p>
+  </div>
+</div>
+  <% @user.posts.each do |post| %>
```

Comment:
- [ ] When clicking on each post should take us to their show page but on your app, the post title is not clickable, when we click on the post nothing happens. Please kindly fix that 🙏🏾